common/common-lang/src/main/java/com/twelvemonkeys/util/regex/WildcardStringParser.java

```diff
@@ -622,7 +622,7 @@ public String toString() {
             buffer.append("\n");
             buffer.append("      Format: <state index>: <character> (<last free state>)");
             buffer.append("\n");
-            buffer.append("      Number of strings parsed: " + totalNumberOfStringsParsed);
+            buffer.append("      Number of strings parsed: ").append(totalNumberOfStringsParsed);
```

Comment:
Doing a regular string `+` inside a stringbuilder append defeats the purpose of having the stringbuilder to begin with. This is the only example found of this.